.github/ionic-issue-bot.yml

```diff
@@ -48,6 +49,24 @@ stale:
     template is fully filled out.
 
 
+    Thank you for using Ionic!
+  close: true
+  lock: true
+  dryRun: false
+
+noReply:
+  days: 30
+  maxIssuesPerRun: 100
+  label: needs reply
+  responseLabel: has reply
```

Comment:
I also had a thought that this could be `triage`, meaning that someone replied to this and thus we should re-triage the issue to determine if the reply was adequate or not, and then route as appropriate. This would avoid the need for yet-another-label.

Reply:
I'm not opposed to it going back into `triage`